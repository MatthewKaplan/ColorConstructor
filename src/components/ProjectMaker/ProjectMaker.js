import React, { Component } from "react";

class ProjectMaker extends Component {
  render() {
    return (
      <div className="project-maker-component">
        <h1>ProjectMaker!</h1>
      </div>
    );
  }
}

export default ProjectMaker;
