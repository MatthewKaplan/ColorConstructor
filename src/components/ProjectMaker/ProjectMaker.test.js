import React from "react";
import { shallow } from "enzyme";
import ProjectMaker from "./ProjectMaker";

describe("ProjectMaker", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ProjectMaker />);
  });

  it("should match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
