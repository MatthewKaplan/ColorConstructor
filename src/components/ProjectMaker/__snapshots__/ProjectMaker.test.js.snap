// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectMaker should match the snapshot 1`] = `ShallowWrapper {}`;
